lib/stdlib/src/peer.erl

```diff
@@ -649,17 +649,19 @@ command_line(Listen, Options) ->
     StartCmd =
         case Listen of
             undefined when map_get(connection, Options) =:= standard_io ->
-                ["-user", atom_to_list(?MODULE)];
```

Comment:
Please break the changes in peer out into a separate commit.

Reply:
Also, please add a testcase to peer_SUITE that tests the `-noshell`/`-user` interaction.

Reply:
I will separate this change to a separate PR. The whole interaction is rather intricate and potentially error-prone, so I need to document it better (or even improve to a point where it's not an issue).